### DISCUSSION TITLE
font size

### DISCUSSION THREAD

**Discussion by shlandrews** [nohal/dashboardsk_pi#72] [Q&A]
is there a way to change font size of dial gauge types if you have increased the system font scaling for high resolution displays? I tried adding body-font: 8 to reduce it in config file, but did not like that. Screenshot attached.
![Screenshot_2024-05-18_17-41-20](https://github.com/nohal/dashboardsk_pi/assets/63079119/c2dd43c7-6b8e-478f-8a5d-1ba3e676cb20)

**Comment by nohal** [comment]
No such setting at the moment, the font size is calculated automatically from the size of the widget. The problem is that the WM does not give a damn crap about that and applies the scaling after we explicitly have told it how big we want the font. The solution should be pretty straightforward, taking the scaling factor into account and asking for a font proportionally smaller in your case. Just to make sure what you set, can you run `gsettings get org.gnome.desktop.interface text-scaling-factor` and confirm it returns a value greater than `1.0`?

**Reply by shlandrews** [reply]
Yes trying to insert a proportional font size into the config file was what I was trying, but unable to get there as yet, text scaling is currently at 1.8

**Reply by nohal** [reply]
Does `gsettings get org.gnome.desktop.interface text-scaling-factor` return `1.8`?

**Reply by shlandrews** [reply]
Yes that's both in the gui settings and what gnome settings returns from cmd line

**Reply by nohal** [reply]
Then if you build the current code and have `glibmm2.4-devel` or whatever it's name on your distribution is, the resulting size of the font used in the gauge should be fine.
